lib/chunks.js

```diff
@@ -475,6 +475,16 @@ module.exports.diffChunks = async ({ coursePath, courseId, courseData, changedFi
     }
   });
 
+  // Check for any deleted questions.
```

Comment:
I ended up implementing deletion for all chunk types; this way everything behaves consistently and there's no special cases.

Reply:
Excellent!